src/services/account/hooks/index.js

```diff
@@ -56,12 +57,14 @@ const trimPassword = (hook) => {
 };
 
 const validatePassword = (hook) => {
+	if (hook.data.userForcedToChangePassword) {
```

Comment:
It is look wrong to skip password validation if in data is set a key.

Reply:
The password is already checked in forcePasswordChange line 6 and 9 .
I had to add this flag because then patch on accounts is performed 
```
	const accountPromise = app.service('accounts')
		.patch(accountId, accountUpdate, params);
```
And in validatePassword method there is a line, which compare passwordVerification with old hashed password. 

I have changed it a bit please take a look